css/locationbar.css

```diff
@@ -17,12 +16,11 @@
 .progressbar {
   z-index: 99;
   display: block;
-  width: 500px;
-  height: 30px;
-  background-image: linear-gradient(110deg, var(--progressbar-color), var(--progressbar-color) 460px, transparent 460px);
-  background-size: 500px 30px;
+  width: 100%;
+  height: 3px;
+  margin-left: -100%;
   position: absolute;
```

Comment:
The intent is to overflow the content. If this is a problem in practice, we could change the design.